spec/defines/resource_upstream_spec.rb

```diff
@@ -242,34 +256,38 @@
                   it {
                     is_expected.to raise_error(Puppet::Error, %r{#{upstream_parameter[:fails][upstreamcontext.to_sym]}})
                   }
+
                   next
                 end
 
                 it {
                   is_expected.to compile.with_all_deps
                 }
+
                 it {
                   is_expected.to contain_concat("#{conf_d_path}/#{title}-upstream.conf").
                     with_mode('0644')
                 }
+
                 it {
                   is_expected.to contain_concat__fragment("#{title}_upstream_header").
                     with_content("# MANAGED BY PUPPET\nupstream #{title} {\n")
                 }
+
                 it {
                   is_expected.to contain_concat__fragment("#{title}_upstream_member_#{params[:members].keys[0]}").
                     with_content("  server #{params[:members].keys[0]}:80;\n")
                 }
 
-                if upstream_parameter[:match] != false
+                if upstream_parameter[:match] == false
```

Comment:
Would it make sense to not swap it but rather write it as
```suggestion
                if upstream_parameter[:match]
```